static/elements/chromedash-process-overview.js

```diff
@@ -140,11 +149,34 @@ class ChromedashProcessOverview extends LitElement {
 
   renderAction(action, stage) {
     const label = action.name;
-    const url = (action.url.replace('{feature_id}', this.feature.id)
-      .replace('{outgoing_stage}', stage.outgoing_stage));
+    const checkCompletion = () => {
+      const url = action.url
+        .replace('{feature_id}', this.feature.id)
+        .replace('{outgoing_stage}', stage.outgoing_stage);
+      const pendingItemsNames = action.prerequisites.filter(
+        itemName => {
+          return !this.progress.hasOwnProperty(itemName);
+        });
+      const pendingItems = pendingItemsNames.map(name => {
+        // return {name, field, stage} for the named item.
+        return this.item_stage_map[name];
+      });
+      if (pendingItems.length > 0) {
+        pendingItems.continueUrl = url;
+        console.info('url', url);
```

Comment:
Missed this comment until now.  Can we require that all conversations are resolved before merging?

Reply:
I kind of like having it not required because it is an extra step.  Usually if there is something missed that is important, we can just do a small follow-up PR.

Reply:
The other part of the equation is that a reviewer should not give approval until after the review feedback that they feel is blocking has been addressed.  In this case, the logging is a minor issue, so I didn't hold my approval waiting for it, which saves one round of review.

Reply:
OK.